samples/msal-browser-samples/VanillaJSTestApp2.0/app/facebook-sample/authConfig.js

```diff
@@ -0,0 +1,71 @@
+// Config object to be passed to Msal on creation
+const msalConfig = {
+    auth: {
+        clientId: "ENTER_CLIENT_ID_HERE",
+        authority: "https://www.facebook.com",
+        knownAuthorities: ["www.facebook.com"],
+        protocolMode: msal.ProtocolMode.OIDC,
+        OIDCOptions: { "serverResponseType": "query", "defaultScopes": ["openid"] },
+        authorityMetadata: '{ "issuer": "https://www.facebook.com", "authorization_endpoint": "https://facebook.com/dialog/oauth/", "token_endpoint": "https://graph.facebook.com/oauth/access_token", "jwks_uri": "https://www.facebook.com/.well-known/oauth/openid/jwks/" }',
+        cache: {
+            cacheLocation: "sessionStorage", // This configures where your cache will be stored
+            storeAuthStateInCookie: false,
+        },
+        system: {
+            allowNativeBroker: false,
```

Comment:
This is good - we should probably default this to `false` in the lib itself if `ProtocolMode` is anything other than `AAD`

Reply:
I can make this change in the other PR #6219